deploy/001_deploy_hattimelockcontroller.js

```diff
@@ -14,9 +14,9 @@ const func = async function (hre) {
     governance = deployer;
   }
 
-  let hatGovernanceDelay = config["timelockDelay"];
```

Comment:
Unrelated to this line but the execute in line 38 shouldn't be done anymore with the new version of time lock controller 0f OZ 4.8.1